testservices/identityservice/v3userpass.go

```diff
@@ -0,0 +1,198 @@
+package identityservice
```

Comment:
It looks like a lot of duplicated code from userpass.go - why not embed UserPass within V3UserPass and reuse it, overriding only where necessary?